Lanchonete/Lanchonete/ Features/Customization/Presentation/CustomizationViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  CustomizationViewController.swift
+//  Lanchonete
+//
+//  Created by vera.dias on 2/9/21.
+//
+
+import UIKit
+
+class CustomizationViewController: UITableViewController, CustomizationView {
+    
+    var presenter: CustomizationViewPresenter?
+    
+    func reloadData() {
+        tableView.reloadData()
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    //MARK: - Section Configuration
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        if let numberOfSections = presenter?.numberOfSections(){
+            return numberOfSections
+        }
+        return -1
+    }
+    
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        if let titleForSection = presenter?.titleForSection(){
+            return titleForSection
+        }
+        return "ERROR"
+        
```

Comment:
This kind of logic is more appropriate for a guard.

Reply:
totally agreed, also changed